ostd/src/mm/page_prop.rs

```diff
@@ -34,6 +34,15 @@ impl PageProperty {
             priv_flags: PrivilegedPageFlags::empty(),
         }
     }
+
+    /// Return if two `PageProperty`s are equal ignoring the ACCESSED and DIRTY flag bits
+    pub fn equal_ignoring_ad(&self, other: &Self) -> bool {
```

Comment:
`equal_ignoring_accessed_dirty`. The `PageFlags` constants are the full words and we should match.